libraries/AP_RPM/AP_RPM_Params.cpp

```diff
@@ -20,11 +20,11 @@ const AP_Param::GroupInfo AP_RPM_Params::var_info[] = {
     // @Param: TYPE
     // @DisplayName: RPM type
     // @Description: What type of RPM sensor is connected
-    // @Values: 0:None,1:Not Used,2:AUXPIN,3:EFI,4:Harmonic Notch,5:ESC Telemetry Motors Bitmask
+    // @Values: 0:None,1:Not Used,2:GPIO,3:EFI,4:Harmonic Notch,5:ESC Telemetry Motors Bitmask
```

Comment:
I agree that AUXPIN was not a good description.  this change is a good one I think.

Reply:
A little lower in this file (lin 59) it says, "Only used on type = PWM." which is out-of-date even for what we have in master.  Perhaps this lower down comment should also be updated to mention "GPIO"

Reply:
fixed